docs/developers/coredev/release.md

```diff
@@ -115,6 +115,23 @@ on the `stdout` if any test fails.
   pytest tools/ --tb=short
   ```
 
+## Update constraints files
+
+`napari` uses a set of constraints files to make sure all dependencies are compatible with each other.
```

Comment:
```suggestion
`napari` uses a set of constraints files to prevent random test crash because of some depndecy update. It allow also for future reprducible build (see linkt to installation instruction here). 
```